formula/src/test/java/com/instacart/formula/FormulaRuntimeTest.kt

```diff
@@ -1249,33 +1254,41 @@ class FormulaRuntimeTest(val runtime: TestableRuntime, val name: String) {
 
     @Test
     fun `inspector events`() {
+        val globalInspector = TestInspector()
+        FormulaPlugins.setPlugin(object : Plugin {
+            override fun inspector(type: KClass<*>): Inspector {
+                return globalInspector
+            }
+        })
```

Comment:
nit: should we have test cases for any possible (global, local) permutation, not only when both are not null?

Reply:
We have some tests for local only, added a new test for global only.